/* @generated */
digraph cfg {
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_1" [label="1: Start void AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString()\nFormals:  this:org.junit.AssumptionViolatedExceptionTest*\nLocals:  $irvar2:void $irvar1:void $irvar0:void cause:java.lang.Throwable* e:org.junit.AssumptionViolatedException*\nAnnotation: <_org.junit.Test> void AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString()() \n  " color=yellow style=filled]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_1" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_4" ;
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_2" [label="2: Exit void AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString() \n  " color=yellow style=filled]
	

"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [label="3:  exceptions sink \n   NULLIFY(&cause); [line 80]\n  NULLIFY(&$irvar1); [line 80]\n  NULLIFY(&$irvar2); [line 80]\n  NULLIFY(&e); [line 80]\n  EXIT_SCOPE(cause,$irvar1,$irvar2,e); [line 80]\n  APPLY_ABSTRACTION; [line 80]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_2" ;
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_4" [label="4:  Call AssumptionViolatedException.<init>(String) \n   n$0=_fun___new(sizeof(t=org.junit.AssumptionViolatedException):org.junit.AssumptionViolatedException*) [line 76]\n  n$1=_fun_AssumptionViolatedException.<init>(String)(n$0:org.junit.AssumptionViolatedException*,\"invalid number\":java.lang.Object*) [line 76]\n  *&e:org.junit.AssumptionViolatedException*=n$0 [line 76]\n  EXIT_SCOPE(n$0,n$1); [line 76]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_4" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_5" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_4" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_5" [label="5:  Call RuntimeException.<init>(String) \n   n$2=_fun___new(sizeof(t=java.lang.RuntimeException):java.lang.RuntimeException*) [line 77]\n  n$3=_fun_RuntimeException.<init>(String)(n$2:java.lang.RuntimeException*,\"cause\":java.lang.Object*) [line 77]\n  *&cause:java.lang.RuntimeException*=n$2 [line 77]\n  EXIT_SCOPE(n$2,n$3); [line 77]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_5" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_6" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_5" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_6" [label="6:  Call Throwable AssumptionViolatedException.initCause(Throwable) \n   n$4=*&e:org.junit.AssumptionViolatedException* [line 78]\n  _=*n$4:org.junit.AssumptionViolatedException*(root org.junit.AssumptionViolatedException) [line 78]\n  n$6=*&cause:java.lang.RuntimeException* [line 78]\n  n$7=_fun_Throwable AssumptionViolatedException.initCause(Throwable)(n$4:org.junit.AssumptionViolatedException*,n$6:java.lang.RuntimeException*) virtual [line 78]\n  *&$irvar0:java.lang.Throwable*=n$7 [line 78]\n  NULLIFY(&$irvar0); [line 78]\n  EXIT_SCOPE(_,n$4,n$6,n$7,$irvar0); [line 78]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_6" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_7" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_6" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_7" [label="7:  Call Throwable AssumptionViolatedException.getCause() \n   n$8=*&e:org.junit.AssumptionViolatedException* [line 79]\n  _=*n$8:org.junit.AssumptionViolatedException*(root org.junit.AssumptionViolatedException) [line 79]\n  n$10=_fun_Throwable AssumptionViolatedException.getCause()(n$8:org.junit.AssumptionViolatedException*) virtual [line 79]\n  *&$irvar1:java.lang.Throwable*=n$10 [line 79]\n  NULLIFY(&e); [line 79]\n  EXIT_SCOPE(_,n$8,n$10,e); [line 79]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_7" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_8" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_7" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_8" [label="8:  Call Matcher CoreMatchers.is(Object) \n   n$11=*&cause:java.lang.RuntimeException* [line 79]\n  n$12=_fun_Matcher CoreMatchers.is(Object)(n$11:java.lang.RuntimeException*) [line 79]\n  *&$irvar2:org.hamcrest.Matcher*=n$12 [line 79]\n  NULLIFY(&cause); [line 79]\n  EXIT_SCOPE(n$11,n$12,cause); [line 79]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_8" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_9" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_8" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_9" [label="9:  Call void MatcherAssert.assertThat(Object,Matcher) \n   n$13=*&$irvar1:java.lang.Throwable* [line 79]\n  n$14=*&$irvar2:org.hamcrest.Matcher* [line 79]\n  n$15=_fun_void MatcherAssert.assertThat(Object,Matcher)(n$13:java.lang.Throwable*,n$14:org.hamcrest.Matcher*) [line 79]\n  NULLIFY(&$irvar1); [line 79]\n  NULLIFY(&$irvar2); [line 79]\n  EXIT_SCOPE(n$13,n$14,n$15,$irvar1,$irvar2); [line 79]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_9" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_10" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_9" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
"org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_10" [label="10:  method_body \n   APPLY_ABSTRACTION; [line 80]\n " shape="box"]
	

	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_10" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_2" ;
	 "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_10" -> "org.junit.AssumptionViolatedExceptionTest.canInitCauseWithInstanceCreatedWithString():void.758fe1dd80dbfc3c4ec9aa869ce50b1e_3" [color="red" ];
}
